server/templates/custom_dc/custom/base.html

```diff
@@ -0,0 +1,113 @@
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-#}
+
+
+{#-
+  Required variables:
+  title: page title
+  main_id: DOM Element ID for the main content block
+  page_id: DOM Element ID for the html element
+
+  Optional variables:
+  locale: string, value for html lang attr
+  subpage_title: string, if specified, will be displayed in the header next to Data Commons
+
+  Blocks to override:
+  head - additional head elements
+  content - main page content
+-#}
+<!DOCTYPE html>
+
+<html lang={{ locale }} id={{ page_id }}>
+
+<head>
+  <title>{{ title }} - Custom Data Commons Commons</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@300;600&family=Roboto&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href={{url_for('static', filename='css/static.min.css', t=config['VERSION'])}}>
+  {% block head %}
+  {% endblock %}
+  {% if OVERRIDE_CSS_PATH %}
+  <link href="{{ OVERRIDE_CSS_PATH }}" rel="stylesheet">
+  {% endif %}
+</head>
+
+<body>
+  <div id="main">
+    <header id="main-header">
+      <nav class="navbar navbar-dark navbar-expand-lg col" id="main-nav">
+        <div class="container-fluid">
+          <div class="navbar-brand">
+            {% if LOGO_PATH %}
+            <div id="main-header-logo">
+                <a href="{{ url_for('static.homepage') }}">
+                  <img src="{{ LOGO_PATH }}" />
+                </a>
+            </div>
+            {% endif %}
+            <a href="{{ url_for('static.homepage') }}">{{ NAME }}</a>
+            {% if subpage_title %}
+            <span>
+              {% if subpage_url %}<a href="{{ subpage_url }}">{% endif %}
+                {{ subpage_title }}
+                {% if subpage_url %}</a>{% endif %}
+            </span>
+            {% endif %}
+            </a>
+          </div>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#dc-main-nav">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse justify-content-md-end" id="dc-main-nav">
+            <ul class="navbar-nav float-md-right">
+                <li class="nav-item dropdown">
+                    <a href="{{ url_for('tools.timeline') }}" id="nav-timeline" class="nav-link-main "><span>Timeline</span></a>
+                </li>
+                <li class="nav-item dropdown">
+                    <a href="{{ url_for('tools.scatter') }}" id="nav-scatter" class="nav-link-main"><span>Scatter</span> </a>
+                </li>
+                <li class="nav-item dropdown">
+                    <a href="{{ url_for('tools.map') }}" id="nav-map" class="nav-link-main "><span>Map</span></a>
```

Comment:
maybe stat var explorer & download tool should be here as defaults

Reply:
I will add those in a follow PR